lib/decors/method_added.rb

```
@@ -5,6 +5,10 @@ module MethodAdded
         module Handler
             private
 
+            METHOD_CALLED_TOO_EARLY_HANDLER = ->(*) {
```

what is the use of this?

---

If you call `decorated_method.bind(...).call` inside `initialize` in the decorator class, it should raise this error.